hack/db-diff.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```

Comment:
Move this to scripts/ maybe and add a short explanation what it does at the top?